package.json

```diff
@@ -20,10 +20,11 @@
     ]
   },
   "dependencies": {
-    "@alicloudfe/components": "^1.1.102",
-    "@alicloudfe/components-console-layout": "1.1.0",
+    "@ant-design/icons": "^4.6.3",
```

Comment:
Remove `@ant-design/icons` package.

Reply:
antd 4.x update icon to @ant-design/icons